src/modules/SystemSettings.rb

```diff
@@ -140,6 +165,8 @@ def SetSysRqKeysEnabled(value)
     publish function: :SetIOScheduler, type: "void (string)"
     publish function: :GetSysRqKeysEnabled, type: "boolean ()"
     publish function: :SetSysRqKeysEnabled, type: "void (boolean)"
+    publish function: :GetAutoConfEnabled, type: "boolean ()"
```

Comment:
The published functions do not correspond to defined methods, isn't it?

Reply:
Grrr, true.